gen3/lib/testData/test1.manifest.g3k/expectedFenceResult.yaml

```diff
@@ -209,6 +218,33 @@ spec:
             echo "${FENCE_PUBLIC_CONFIG:-""}" > "/var/www/fence/fence-config-public.yaml"
             python /var/www/fence/yaml_merge.py /var/www/fence/fence-config-public.yaml /var/www/fence/fence-config-secret.yaml > /var/www/fence/fence-config.yaml
             bash /fence/dockerrun.bash && if [[ -f /dockerrun.sh ]]; then /dockerrun.sh; fi
+      - name: nginx-prometheus-exporter-wrapper
+        image: quay.io/cdis/nginx-prometheus-exporter-wrapper:latest
```

Comment:
Provide a manifest image key with a fallback.
Merge the new Dockerfile first, tag the image, then submit a separate PR with the k8s yaml files that reference the tagged image as a fallback